contracts/EthTranchePricing.sol

```diff
@@ -48,9 +48,7 @@ contract EthTranchePricing is PricingStrategy, Ownable {
   /// @param _tranches uint[] tranches Pairs of (start amount, price)
   function EthTranchePricing(uint[] _tranches) {
     // Need to have tuples, length check
-    if(_tranches.length % 2 == 1 || _tranches.length >= MAX_TRANCHES*2) {
-      throw;
-    }
+    require((_tranches.length % 2 == 0) && (_tranches.length < MAX_TRANCHES*2));
```

Comment:
Mikko, this is well translated from "if" to "require()", however, should be really do "_tranches.length < MAX_TRANCHES*2" instead of "_tranches.length <= MAX_TRANCHES*2"?

Reply:
Good catch, I've changed condition to _tranches.length <= MAX_TRANCHES * 2.